src/test/controller/UserController.test.ts

```diff
@@ -286,29 +286,14 @@ describe('Unit tests for UserController', function () {
               UserRepository
             );
             await expect(
-              userRepository.findOneOrFail({ id: expectedUser.id })
+              userRepository.findOneOrFail({ id: userToDelete.id })
             ).rejects.toThrowError();
             done();
           }
         );
       });
   });
 
-  test('DELETE /users/:id - delete non-existent user should not fail', async (done) => {
```

Comment:
I removed this test because it's no longer valid or possible to test now that we have the auth protection! This is because we have two scenarios:

1. Non-existent user is the authenticated user
2. Non-existent user is not the authenticated user

If 1 is the case, then we'll either save or not save the user before making the call to the endpoint. If we save, then the user is no longer nonexistent, but if we don't save, it'll throw an "EntityNotFound" when we check for the user using the firebaseId which is not what we're trying to test here (already tested)!

If 2 is the case, then a "ForbiddenError" will be thrown, which again is not what we're trying to test here (already tested)!

Reply:
Excellent points, thanks for explaining this! Great job for looking into other parts of our code that are affected with your changes like this test case